Makefile

```diff
@@ -8,12 +8,19 @@ include ${BSIM_BASE_PATH}/common/pre.make.inc
 
 SRCS:=$(shell find src/ -name "*.c")
 
+dummy:
```

Comment:
Please don't create a new default rule. (The default rule is compile. This file should not add any rule anyhow)
Is there any reason to not use the type of check I proposed?

Reply:
Without creating a rule, I got an error

Reply:
Using what I wrote?

Reply:
Arg. I indented the error message. That is why it went wrong